packages/builtin_test/_test.pony

```diff
@@ -864,14 +865,27 @@ class iso _TestStringFromIsoArray is UnitTest
     h.assert_eq[String](s, "foo")
     h.assert_eq[USize](s.size(), 3)
     h.assert_true((s.space() == 3) or s.is_null_terminated())
-/*
-    let s2 = String.from_iso_array(recover
+
+    let s2 = recover val String.from_iso_array(recover
+      ['1', '1', '1', '1', '1', '1', '1', '1']
+    end) end
+    h.assert_eq[String](s2, "11111111")
+    h.assert_eq[USize](s2.size(), 8)
+    h.assert_true((s2.space() == 8) or s2.is_null_terminated())
```

Comment:
@jemc Because the test relies on the preallocation behavior of array literals, and I didn't think these tests should depend on that behavior.

I guess it would make more sense to do:
```pony
h.assert_true((s2.space() == 8) != s2.is_null_terminated())
```